exercises/palindrome-products/Example.fs

```diff
@@ -11,7 +11,7 @@ let isPalindrome n =
 
 let palindrome predicate compare startValue minFactor maxFactor = 
     if minFactor > maxFactor then
-        None
+        failwith "min must be <= max"
```

Comment:
```suggestion
        invalidArg "min must be <= max"
```

As we are comparing parameters here, it is probably slightly more idiomatic to throw an `ArgumentException`, for which the `invalidArg` function can be used: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/exception-handling/the-invalidarg-function

Reply:
Sure, invalidArg wants the argument as first parameter it could be the tuple (min, max)